drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -289,6 +289,9 @@ static int lis2dw12_init(const struct device *dev)
 		return ret;
 	}
 
+	LOG_INF("bandwidth filter is %d", (int)cfg->bw_filt);
```

Comment:
LOG_INF seems a bit much; why did you select this instead of LOG_DBG like the other calls in this same function?